src/hooks/useData.js

```diff
@@ -1,29 +1,19 @@
 import { useState, useEffect } from "react";
 
-const useData = (fileName, mapFunction) => {
+const useData = (service, fileName, mapFunction) => {
```

Comment:
Is this hook being used? i cannot find where are you using it

Reply:
I was trying to implement your suggestion from last week (alternating between mock and real data), but I bumped into too many issues and decided to go for full API integration. I'll delete useData.

Reply:
I get it haha now that i've read the api documentation I can totally understand the headache it would be to make this work